08LessonEight/in-class-project/css/homework-style.css

```diff
@@ -0,0 +1,138 @@
+* {
+  font-family:sans-serif;
+  color: #636363;
+  font-weight: 100;
+}
+
+header {
+  width: 40vw;
+  margin:0 auto;
+  margin-bottom: 3%;
+  font-size: 12px;
+  text-align: center;
+}
+
+main {
+  width: 70%;
+  margin: 0 14% 0 16%;
+  font-size: 10px;
+
+}
+
+h3 {
+  font-size: 14px;
+  margin: 0;
+}
+
+a {
+  float: right;
+  color: #77c95d;
+  padding-right: 20px;
+}
+
+#leftpanel {
+  background-color: #f2f2f2;
+  height:55vh;
+  width: 35vw;
+  float: left;
+  text-align: left;
+}
+
+#leftpanel img {
+  width:93%;
+  margin: 10px 12px 0 12px;
+}
+
+#leftpanel * {
+margin: 10px 0 0 10px;
+}
+
+#rightpanel {
+  height:55vh;
+  width: 33vw;
+  float: left;
+  overflow: hidden;
+}
+
+.rightbox {
+  background-color: #f2f2f2;
+  height:31%;
+  width: 100%;
+  margin: 0 0 10px 10px;
+  box-sizing: border-box;
+  position: relative;
+}
+
+.rightbox img {
+  height:80%;
+  float:left;
+  margin:8px 10px 12px 10px;
+}
+
+.rightboxtext {
+  padding-top: 12px;
+  width:95%;
+  text-align: left;
+}
+
+#middle {
+  clear: both;
+  width:100%;
+  height: 100%;
+  /*margin:auto;*/
+}
+
+br {
+  clear: both;
+}
+
+#foot {
+  border-top:1px solid #dddddd;
+  border-bottom: 1px solid #dddddd;
+  width: 99%;
+  height: 120px;
+  overflow:overlay;
+  margin-top: 3%;
+  margin-right: 0;
+  margin-bottom: 30px;
+}
+
+.footbox {
+  width:33%;
+  height: 70%;
+  float: left;
+  padding-left: 20px;
+  margin: 17px auto;
+  text-align: left;
+  box-sizing: border-box;
+}
+
+#footbox1 {
+  border-right-color: #dddddd;
+  border-right-width:thin;
+  border-right-style: solid;
+}
+
```

Comment:
these two IDs are redundant, you could try making a class and applying that to the elements desired.